api/src/main/java/org/openmrs/api/db/hibernate/ChainingInterceptor.java

```diff
@@ -230,16 +233,6 @@ public void beforeTransactionCompletion(Transaction tx) {
 		}
 	}
 	
-	// passes the sql returned from each previous onPrepareStatement onto the next
-	@Override
-	public String onPrepareStatement(String sql) {
```

Comment:
Was this removed because it is not used anywhere?

Reply:
It is because it was removed without a replacement, I left a longer explanation on the jira doc, but yea essentially this was deprecated -https://docs.jboss.org/hibernate/orm/5.3/javadocs/org/hibernate/Interceptor.html#onPrepareStatement(java.lang.String)

It mentions if you need to use similar behaviour to use `StatementInspector`, but seemed to me this wouldn't work as a replacement in this case as we are chaining the `Interceptor` interface